redfish-core/include/redfish.hpp

```diff
@@ -121,7 +121,6 @@ class RedfishService
         requestRoutesStorageController(app);
         requestRoutesDrive(app);
         requestRoutesCable(app);
-        requestRoutesCableCollection(app);
```

Comment:
Why remove this ?

Reply:
During upstream activities (for Port/PortCollection), Ed had suggested to combine those route calls.
https://gerrit.openbmc.org/c/openbmc/bmcweb/+/61097/comment/ddb59a09_86072aa4/

>Ed Tanous
>Please just combined these two into one. There's no need to have multiple requestRoutes. i realize we separate >this in other places, but that was because of refactoring.

So, I did it, but as a part of this, maybe we can go back .... until the next refactor.